api/lib/validation/schemaValidator.js

```diff
@@ -0,0 +1,183 @@
+'use-strict'
```

Comment:
This was to handle the case where black friday started and you wanted a range from 1 - 10 but you knew you needed to start at nothing less than 5.
Its really initial_min_instance_count.